packages/fieldset/src/LionFieldset.js

```diff
@@ -263,16 +317,20 @@ export class LionFieldset extends FormRegistrarMixin(
     }
   }
 
-  _updateFocusedClass() {
-    this.classList[this.touched ? 'add' : 'remove']('state-focused');
+  _syncFocused() {
+    this.focused = this._anyFormElementHas('focused');
```

Comment:
```suggestion
    this.focused = true;
````

Reply:
yeah won't work as the focusin event has `composed: true` 😭 
so we need this extra check here